.htaccess

```diff
@@ -1,6 +1,6 @@
 Options -MultiViews
 RewriteEngine On
-RewriteBase /mother_care/
+RewriteBase /mother_care/ 
```

Comment:
@Shubham2801 
please add this to your gitignore.
recommit without this file's changes.